testrunner-toolkit/cypress/cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
```

Comment:
Can you provide an example? If not, then please add a link that will go to the Cypress docs on how to use this